libmcount/record.c

```diff
@@ -507,6 +511,7 @@ static int record_ret_stack(struct mcount_thread_data *mtdp,
 	if (type == UFTRACE_EXIT)
 		timestamp = mrstack->end_time;
 
+	/* TODO: need to explain in comments to show the behavior */
```

Comment:
I thought it's almost done but I'm not sure how to write the record properly here.
@namhyung Could you please explain about the below code?